### DISCUSSION TITLE
More 'blue' icons

### DISCUSSION THREAD

**Discussion by ghost** [moudey/Shell#130] [Ideas]
**First of all, this thing is phenomenal. No - not phenomenal. Much beyond that. I have a Windows 11 and a Windows 10 laptop - installed this into both, and the change in look and feel (and usability ofc) is dramatic. I mean, just look at it - so beautiful...**

![image](https://user-images.githubusercontent.com/85873175/214481877-866fbb2e-a3e4-4e7f-b98e-e88ee287d574.png)

**However, the icons sometimes are fairly peculiar. For example, have a look at this Cortana icon.**
![image](https://user-images.githubusercontent.com/85873175/214482013-4efbadf8-f345-44a8-8df9-f626136d59e4.png)

She ain't just a circle 😂
How about this mockup (made in MS Paint xD)
![image](https://user-images.githubusercontent.com/85873175/214482270-c75abea6-91f5-4313-b2b9-aa44362373b4.png)

**Comment by moudey** [comment]
SVG version will be included
![cortana](https://user-images.githubusercontent.com/39198/214487815-efdb3d7d-a04e-411d-a7bc-11cc0ac6211c.svg)

**Reply by ghost** [reply]
It looks good, but please reduce the stroke width otherwise it will be inconsistent with the other icons...

**Comment by ghost** [comment]
A bit of an off-topic issue here...
![image](https://user-images.githubusercontent.com/85873175/214507364-e2360b38-1998-40ef-bc5b-4ec117dd9646.png)

The phrase "File Manage" somehow seems odd and incomplete. Why not change it to 'File Manage _ment_ '?
